lib/core/services/hashing_service.dart

```diff
@@ -21,13 +22,16 @@ class HashingService {
   ) async {
     try {
       final parts = hashedPassword.split(':');
-      if (parts.length != 2) return false;
+      const int two = 2;
```

Comment:
its bullshit. need to add meaningful names, like in this case number of parts of hashed password (salt and hash). lets call it numberOfParts